index.html

```diff
@@ -30,16 +50,20 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
```

Comment:
make sure to use flexbox to its full potential. Aka use the display: flex / flex-direction/ aligh-items & justify content BEFORE going to padding & margin.

Always check to make sure the CSS change you implemented is actually having the desired effect